app/javascript/components/mapping/MappingForm.jsx

```diff
@@ -85,7 +85,7 @@ const MappingForm = () => {
    * out the "use case" input
    */
   const handleUseCaseBlur = () => {
-    if (!validURL(useCase)) {
+    if (!_.isEmpty(useCase) && !validURL(useCase)){
```

Comment:
An empty value isn't a valid URL anyway, so the `isEmpty` check is unnecessary here.

Reply:
Correct, but I don't want the validator to even run if the user didn't fill the input.

Let's say the user doesn't want to put a URL for the use case input, and he/she uses the keyboard, it will inevitably go through the input without typing and with the logic like `if (!validURL(useCase)) {` there will be an error prompt saying `It must be a valid URL!` but the user didn't type anything, so... 😒.

That's why I wanted to also validate the emptiness of the field value. And this way it does not prompt the user, unless he/she types something that's not a URL.